modules/end-user-guide/pages/git-credential-store.adoc

```diff
@@ -1,25 +1,89 @@
-:navtitle: Git credential store
+:navtitle: Creating a {orch-name} Secret for a Git credentials store
 :keywords: user-guide, configuring, user, secrets
 :page-aliases: 
 
-[id="mounting-a-git-credential-store-into-a-workspace-container_{context}"]
-= Mounting a Git credentials store into a workspace container
+[id="mounting-a-git-credential-store-into-workspace-containers_{context}"]
+= Creating a {orch-name} Secret for a Git credentials store
 
-This section describes how to mount Git credentials store as secret from the user's {orch-namespace} into the file in single-workspace or multiple-workspace containers of {prod-short}.
+This section describes how to create a Secret containing a Git credentials store in the user's {orch-namespace}. The Git credentials store specified in the Secret is mounted as a file in the {devworkspace}'s containers.
+
+When mounting the Secret, a Git configuration file with the path to the mounted Git credentials store is automatically configured and mounted to the {devworkspace} containers at `/etc/gitconfig`.
 
 .Prerequisites
 
 * A running instance of {prod-short}.
+* You have CLI or GUI access to the {orch-name} cluster of your organization's {prod-short} instance.
+
+ifeval::["{project-context}" == "che"]
+** For CLI users: `{orch-cli}` is installed in the operating system you are using. See link:https://kubernetes.io/docs/tasks/tools/#kubectl[Install Tools: kubectl].
+endif::[]
+
+ifeval::["{project-context}" == "crw"]
+** For CLI users: `{orch-cli}` is installed in the operating system you are using. See link:https://docs.openshift.com/container-platform/4.10/cli_reference/openshift_cli/getting-started-cli.html#installing-openshift-cli[Installing the OpenShift CLI].
+endif::[]
+
+* For CLI users: link:https://www.gnu.org/software/coreutils/base64[`base64`] is installed in the operating system you are using. For Mac users, `base64` is available (as part of `coreutils`) on link:https://formulae.brew.sh/formula/coreutils[homebrew].
```

Comment:
@dkwon17, are you able to use `cat`, `echo`, and tr` on Mac?

Reply:
@max-cx yes, those should be available out of the box